### ISSUE TITLE
Unable to create templates in the addon module

### ISSUE THREAD

**Issue by Bharat-B** [acronis/acronis-cyber-cloud-whmcs#2]
```
{"error":{"code":"23000","message":"SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`********`.`acroniscloud_service_template_applications`, CONSTRAINT `acroniscloud_service_template_applications_template_id_foreign` FOREIGN KEY (`template_id`) REFERENCES `acronisc...) (SQL: insert into `acroniscloud_service_template_applications` (`type`, `status`, `editions`, `template_id`) values (backup, active, [{\"name\":\"standard\",\"status\":\"active\"}], 2259973))"}}

```

MariaDB [********]> describe acroniscloud_service_template_applications;
+-------------+---------------------------+------+-----+---------+----------------+
| Field       | Type                      | Null | Key | Default | Extra          |
+-------------+---------------------------+------+-----+---------+----------------+
| id          | int(10) unsigned          | NO   | PRI | NULL    | auto_increment |
| type        | varchar(255)              | NO   |     | NULL    |                |
| editions    | varchar(1024)             | NO   |     | NULL    |                |
| status      | enum('active','inactive') | NO   |     | NULL    |                |
| template_id | int(10) unsigned          | NO   | MUL | NULL    |                |
+-------------+---------------------------+------+-----+---------+----------------+
5 rows in set (0.001 sec)

MariaDB [********]> SELECT
    -> TABLE_NAME,COLUMN_NAME,CONSTRAINT_NAME, REFERENCED_TABLE_NAME,REFERENCED_COLUMN_NAME
    -> FROM
    -> INFORMATION_SCHEMA.KEY_COLUMN_USAGE
    -> WHERE
    -> REFERENCED_TABLE_SCHEMA = 'lsipuspl_w204s24' AND
    -> REFERENCED_TABLE_NAME  = 'acroniscloud_service_template_applications';
+----------------------------------------------+-------------------------+---------------------------------+--------------------------------------------+------------------------+
| TABLE_NAME                                   | COLUMN_NAME             | CONSTRAINT_NAME                 | REFERENCED_TABLE_NAME                      | REFERENCED_COLUMN_NAME |
+----------------------------------------------+-------------------------+---------------------------------+--------------------------------------------+------------------------+
| acroniscloud_service_template_offering_items | template_application_id | template_application_id_foreign | acroniscloud_service_template_applications | id                     |
+----------------------------------------------+-------------------------+---------------------------------+--------------------------------------------+------------------------+
1 row in set (0.016 sec)

MariaDB [***********]> describe acroniscloud_service_template_offering_items;
+-------------------------+---------------------------+------+-----+---------+----------------+
| Field                   | Type                      | Null | Key | Default | Extra          |
+-------------------------+---------------------------+------+-----+---------+----------------+
| id                      | int(10) unsigned          | NO   | PRI | NULL    | auto_increment |
| name                    | varchar(255)              | NO   |     | NULL    |                |
| infra_id                | varchar(36)               | YES  |     | NULL    |                |
| status                  | enum('active','inactive') | NO   |     | NULL    |                |
| measurement_unit        | varchar(255)              | NO   |     | NULL    |                |
| quota_value             | bigint(20) unsigned       | YES  |     | NULL    |                |
| template_application_id | int(10) unsigned          | NO   | MUL | NULL    |                |
+-------------------------+---------------------------+------+-----+---------+----------------+
7 rows in set (0.001 sec)

MariaDB [********]> select * from acroniscloud_service_template_offering_items\G;

Empty set (0.000 sec)

ERROR: No query specified
```

**Comment by merx3** [comment]
Hello Bharat,

This is a known issue related to our logs that we're still in the process of fixing. 
For the db queries log option ("Queries to WHMCS database"), this should be fixed in the latest version of the module in the WHMCS marketplace. Please inform me if that's not the case for you.
For the API calls to Acronis log option ("API calls to Acronis Cyber Cloud"), it's still not fixed yet, so until there's an update with the fix, please just disable it. Sorry for the inconvenience.

**Comment by merx3** [comment]
This issue was fixed with the release of version 2.2.0-175
